libshaderc/src/shaderc_test.cc

```diff
@@ -52,6 +52,14 @@ TEST(Init, MultipleThreadsCalling) {
   shaderc_compiler_release(compiler3);
 }
 
+TEST(Init, SPVVersion) {
+  unsigned int version = 0;
+  unsigned int revision = 0;
+  shaderc_get_spv_version(&version, &revision);
+  EXPECT_EQ(version, spv::Version);
```

Comment:
Ah, I didn't realize order mattered (never used this test framework before). I'll send another update later this morning.